File: app-miyou/src/main/java/com/miyou/app/domain/monitoring/model/MetricsRollup.java
package com.miyou.app.domain.monitoring.model;

import java.time.Instant;

public record MetricsRollup(
	Instant bucketStart,
	com.miyou.app.domain.monitoring.model.MetricsGranularity granularity,
	long requestCount,
	long totalTokens,
	long totalDurationMillis,
	double avgResponseMillis
) {
}

File: domain/src/main/kotlin/com/miyou/app/domain/monitoring/model/MetricsRollup.kt
package com.miyou.app.domain.monitoring.model

import java.time.Instant

data class MetricsRollup(
    val bucketStart: Instant,
    val granularity: MetricsGranularity,
    val requestCount: Long,
    val totalTokens: Long,
    val totalDurationMillis: Long,
    val avgResponseMillis: Double,
)
